plugins/janus_audiobridge.c

```diff
@@ -1070,6 +1084,15 @@ int janus_audiobridge_init(janus_callbacks *callback, const char *config_path) {
 			audiobridge->audiolevel_ext = TRUE;
 			if(audiolevel_ext != NULL && audiolevel_ext->value != NULL)
 				audiobridge->audiolevel_ext = janus_is_true(audiolevel_ext->value);
+			audiobridge->audiolevel_event = FALSE;
+            if(audiolevel_event != NULL && audiolevel_event->value != NULL)
+            	audiobridge->audiolevel_event = janus_is_true(audiolevel_event->value);
+            audiobridge->audio_active_packets = 100;
+            if(audio_active_packets != NULL && audio_active_packets->value != NULL)
+                audiobridge->audio_active_packets = atol(audio_active_packets->value);
```

Comment:
Why `atol` for those two values, if they're both `int`? `atoi` is the right method for that.